packages/components/src/shortened-number/formattedNumber.tsx

```diff
@@ -0,0 +1,8 @@
+const FORMATTER = new Intl.NumberFormat();
+function formatNumber( number: number | null ) {
```

Comment:
```suggestion
export default function formattedNumber( number: number | null ) {
```

Could we directly export this function instead?